R/elevation.R

```diff
@@ -12,17 +12,13 @@
 #' @return A modified version of the input `dat` with an additional `z_` column
 #' appended to the vertices.
 #' @family transform
+#'
 #' @examples
 #' # Bounding box of "omaha nebraska":
 #' bb <- c (-96.2682, 41.190585, -95.871141, 41.3930618)
-#' query <- opq (bb)
-#' query <- add_osm_feature (query, key = "highway")
-#' # Equivalent to:
-#' \dontrun{
-#' query <- opq ("omaha nebraska") |>
+#' query <- opq (bb) |>
```

Comment:
```suggestion
#' query <- opq ("omaha nebraska") |>
```
I think this, and all others, are clearer if the not-run parts just have the place names, because that's how I imagine this function mostly gets used (including by me).